### PR TITLE
MS MARCO documents on ElasticSearch

### PR DESCRIPTION
**Pull request by HangCui0510**
Default BufferLimit
Screen Shot 2020-05-04 at 11 17 35 AM
Increase BufferLimit:
Screen Shot 2020-05-04 at 11 37 26 PM

### CODE REVIEW

**Comment by HangCui0510** [comment]
@lintool , please review

File: src/main/python/run_es_regression.py [hunk 1]
```diff
@@ -1,13 +1,10 @@
 # -*- coding: utf-8 -*-
 '''
 Anserini: A Lucene toolkit for replicable information retrieval research
-
```
Comment (lintool) [inline src/main/python/run_es_regression.py:4]: can you not changes these lines?

File: src/main/python/run_es_regression.py [hunk 2]
```diff
@@ -155,9 +163,10 @@ def evaluate(self, collection):
         if collection == 'robust04': expected = 0.2531
         elif collection == 'msmarco-passage': expected = 0.1956
         elif collection == 'core18': expected = 0.2495
+        elif collection == 'msmarco-doc': expected = 0.2308
         else: raise Exception('Unknown collection: {}'.format(collection))
 
-        if math.isclose(ap, expected): logger.info('[SUCESS] {} MAP verified as expected!'.format(ap))
+        if math.isclose(ap, expected): logger.info('[SUCCESS] {} MAP verified as expected!'.format(ap))
```
Comment (lintool) [inline src/main/python/run_es_regression.py:169]: and this?

**Comment by eiston** [comment]
@lintool @HangCui0510 I have replicated this pull request and have run the end to end regression test for msmarco-passage, I can confirm that this pull request works.

**Comment by lintool** [comment]
I merged in latest master, but build appears to be broken now?

**Comment by eiston** [comment]
> I merged in latest master, but build appears to be broken now?

Sorry, I forgot to recompile java, I can see the compilation error now.

**Comment by HangCui0510** [comment]
should work now, please check my new PR
Screen Shot 2020-05-05 at 2 58 15 PM

**Comment by lintool** [comment]
superseded by #1151

**Pull request closed without merging**